src/traces/sunburst/attributes.js

```diff
@@ -203,6 +203,7 @@ module.exports = {
             'this option won\'t have any effect and `insidetextfont` would be used.'
         ].join(' ')
     }),
+    rotation: pieAttrs.rotation,
```

Comment:
I think we need to tweak a number of keys for this new attribute.
The default angle of `pie` is zero which starts at 12:00. `sunburst` on the other hand starts at 3:00.
One may try changing the `editType` from `calc` to `plot`.
So please consider necessary changes like this:
```js
rotation: extendFlat({}, pieAttrs.rotation, {
  editType: 'plot',
   dflt: 90, // or maybe -90?
   description: '< please adapt pie description for sunburst considering its default value and look here >'
}
```

Reply:
@thierryVergult 
Instead of above please pick this https://github.com/plotly/plotly.js/commit/2af75a8179b177daf603d8d004470249db5bce7b commit and this would be ready for final review.

Reply:
Thanks. I copied the changes (definition of the rotation attribute in my local branch).

The extra rotation is counter clockwise. Should I change that to clockwise ? (what is common in other traces in plotly?)

Reply:
Thanks.
The default direction of `pie` is also `counterclockwise`. 
So there is no problem with counter clockwise rotation in `sunburst`.